### PR TITLE
docs(plutono): How to create dashboards

### PR DESCRIPTION
**Pull request by richardtief**
To be added to the operations bundles.

### CODE REVIEW

File: plutono/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ The plugin works by factory default with anonymous access enabled. If you use th
 
 Dashboards are selected from `ConfigMaps` across namespaces. The plugin searches for `ConfigMaps` with the label `plutono-dashboard: "true"` and imports them into Plutono. The `ConfigMap` must contain a key like `my-dashboard.json` with the dashboard JSON content. [Example](https://github.com/cloudoperators/greenhouse-extensions/blob/main/plutono/README.md#example-dashboard-config)
 
+A guide how to create dashboards can be found [here](https://github.com/cloudoperators/greenhouse-extensions/blob/main/plutono/README.md#create-a-dashboard).
```
Comment (viennaa) [inline plutono/README.md:26]:
```suggestion
A guide how to create dashboards can be found [here](#dashboard).
```

File: plutono/README.md [hunk 2]
```diff
@@ -332,12 +334,39 @@ dashboards:
       datasource:
       - name: DS_PROMETHEUS
         value: Prometheus
-      - name: DS_LOKI
-        value: Loki
     local-dashboard:
       url: https://raw.githubusercontent.com/user/repository/master/dashboards/dashboard.json
 ```
 
+## Create a dashboard
```
Comment (viennaa) [inline plutono/README.md:341]:
```suggestion
## Create a dashboard {#dashboard}
```

File: plutono/README.md [hunk 3]
```diff
@@ -332,12 +334,39 @@ dashboards:
...
+   - Data source options: Select the data source **Type** e.g. `Prometheus`.
+   - Click **Update**.
+   - Go back.
+
+5. Develop your panels.
+   - On the **Edit panel** view, choose your desired **Visualization**.
+   - Select the datasource variable you just created.
+   - Write or construct a query in the query language of your data source.
+   - Move and resize the panels as needed.
+6. Optional add a **tag** to the dashboard to easy group them.
```
Comment (viennaa) [inline plutono/README.md:360]:
```suggestion
6. Optionally add a **tag** to the dashboard to make grouping easier.
```

**Review by viennaa — CHANGES_REQUESTED** [review]
Cool, some small things.

File: plutono/README.md (hunk 1 above)
Comment (ibakshay) [inline plutono/README.md:26]:
```suggestion
A guide on how to create dashboards can be found [here](#dashboard).
```

**Review by ibakshay — COMMENTED** [review]
very comprehensive and easy to follow documentation! 🚀

File: plutono/README.md [hunk 4]
```diff
@@ -332,12 +334,39 @@ dashboards:
...
+   - Go to **Dashboard settings**.
+   - In the **General** section, add a **Tag**.
+7. Click **Save**. Note that the dashboard is saved in the browser's local storage.
+8. Export the dashboard.
+   - Go to **Dashboard settings**.
+   - Click **JSON Model**.
+   - Copy the JSON model.
+   - Go to your Github repository and create a new JSON file in the `dashboards` directory.
```
[the remark this answers is not in the document]
Comment (richardtief) [reply plutono/README.md:368]: `.json`

File: plutono/README.md [hunk 5]
```diff
@@ -23,6 +23,8 @@ The plugin works by factory default with anonymous access enabled. If you use th
 
 Dashboards are selected from `ConfigMaps` across namespaces. The plugin searches for `ConfigMaps` with the label `plutono-dashboard: "true"` and imports them into Plutono. The `ConfigMap` must contain a key like `my-dashboard.json` with the dashboard JSON content. [Example](https://github.com/cloudoperators/greenhouse-extensions/blob/main/plutono/README.md#example-dashboard-config)
 
+A guide how to create dashboards can be found [here](#dashboard).
```
Comment (richardtief) [inline plutono/README.md:26]:
```suggestion
A guide on how to create dashboards can be found [here](#dashboard).
```

**Review by ibakshay — APPROVED** [review]
🚀

**Pull request merged**